lib/identity/stateTransitions/identityCreateTransition/validateIdentityCreateSTDataFactory.js

```diff
@@ -24,6 +25,11 @@ function validateIdentityCreateSTDataFactory(dataProvider) {
   async function validateIdentityCreateSTData(identityCreateTransition) {
     const result = new ValidationResult();
 
+    // check if public keys are valid
+    result.merge(
```

Comment:
We can validate it without state so we could move it to the validate structure function.